db/migrate/20201215095226_change_address_postcode_to_postcode.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeAddressPostcodeToPostcode < ActiveRecord::Migration[6.0]
+  def change
+    rename_column :placement_suitabilities, :address_postcode, :postcode
```

Comment:
Why this change? Wouldn't it be better to keep all the address fields with consistent naming?

Reply:
It was just to make it easier to use the shared example. We could update the naming on the placement_needs to use the address prefix, I'm easy...